lib/google_places/spot.rb

```diff
@@ -398,8 +425,10 @@ def self.request(method, multipage_request, exclude, options)
 
     def self.multi_pages_request(method, multipage_request, options)
       begin
+        results_key = options.delete(:results_key) || 'results'
```

Comment:
Maybe skip multi_pages_request all together? Willing to accept a new direction here.